src/seahawk/seahawk_deck/thrust.py

```diff
@@ -39,23 +41,21 @@ class Thrust(Node):
     def __init__(self):
         """Initialize this node"""
         super().__init__('thrust')
+
+        self.motor_config = [
+            [0, 0, 0, 0, 0.7071, 0.7071, -0.7071, -0.7071], 
+            [0, 0, 0, 0, -0.7071, 0.7071, -0.7071, 0.7071], 
+            [1, 1, 1, 1, 0, 0, 0, 0],
+            [-.19, -.19, .19, .19, -.105, -.105, .105, .105],
+            [-.12, .12, -.12, .12, -.15, .15, -.15, .15],
+            [-.047,-.047,-.047, -.047, 0.038, 0.038, 0.038, 0.038]
+        ]
```

Comment:
Can you add a brief code comment describing what exactly this is, and how we got it? We can save specifics for markdown documentation (should probably write that too!)